plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/cache/TestCachingHiveMetastoreWithQueryRunner.java

```diff
@@ -156,6 +156,17 @@ public void testIllegalFlushHiveMetastoreCacheProcedureCalls()
 
         assertThatThrownBy(() -> getQueryRunner().execute("CALL system.flush_metadata_cache(schema_name => 'dummy_schema', table_name => 'dummy_table', partition_column => ARRAY['dummy_partition'])"))
                 .hasMessage("Parameters partition_column and partition_value should have same length");
+
+        assertThatThrownBy(
+                () -> getQueryRunner().execute("CALL system.flush_metadata_cache(" +
```

Comment:
You could use `com.google.common.collect.Lists#cartesianProduct(java.util.List<? extends java.util.List<? extends B>>)` for covering all the failure use-cases for `('partition_column', 'partition_value')` x `('partition_columns', 'partition_values')`.